### PR TITLE
applications: nrf_desktop: Update info about thread and memory

### PR DESCRIPTION
**Pull request by pdunaj**
Jira:DESK-825

Signed-off-by: Pawel Dunaj <pawel.dunaj@nordicsemi.no>

### CODE REVIEW

File: applications/nrf_desktop/README.rst [hunk 1]
```diff
@@ -88,8 +88,34 @@ Thread usage
...
+Memory allocation
+-----------------
 
+Most of the memory resources used by the application are statically allocated.
+
+The application uses dynamic allocation for creating the event manager events.
+More about events allocation can be read in :ref:`event_manager`.
+
+.. note::
+    When configuring HEAP remember to set both CONFIG_HEAP_MEM_POOL_SIZE and CONFIG_HEAP_MEM_POOL_MIN_SIZE to match typical event size and the system needs.
```
Comment (MarekPieta) [inline applications/nrf_desktop/README.rst]: ``CONFIG_HEAP_MEM_POOL_SIZE ``

File: applications/nrf_desktop/README.rst [hunk 2]
```diff
@@ -88,8 +88,34 @@ Thread usage
 ------------
 
 The application limits the number of threads used to the minimum and does not use user-space threads.
+
+Threads running are:
+    * System related threads:
+        * the idle thread,
+	* the system workqueue thread,
+	* the logger thread (on debug build types),
```
Comment (MarekPieta) [inline applications/nrf_desktop/README.rst]: There is `shell_rtt` thread too

File: applications/nrf_desktop/README.rst (hunk 1 above)
Comment (pdunaj) [reply applications/nrf_desktop/README.rst]: done

File: applications/nrf_desktop/README.rst (hunk 2 above)
Comment (pdunaj) [reply applications/nrf_desktop/README.rst]: done

**Comment by b-gent** [comment]
looks good!

**Pull request merged**